0008-rework-inspect/README.md

```diff
@@ -0,0 +1,414 @@
+
+Viewing manifests and values files after Zarf variable templating would be useful for both creators and deployers. Catching a mistake in templating early can reduce cycle time. A Helm template is almost instant, whereas create + deploy could take several minutes to hours.
+
+A user can achieve a similar effect to `zarf package inspect manifests` by decompressing their package and running `helm template` on their chart. Not only is this a poor UX, but the `helm template` may fail depending on where Zarf variable templating is used within the chart.
+
+Features relating to these problems have been highly requested in recent months:
+- Request in Kubernetes Slack - https://kubernetes.slack.com/archives/C03B6BJAUJ3/p1730229638367829
+- An issue has been created for this - https://github.com/zarf-dev/zarf/issues/2631
+- Defense Unicorns, an organization that relies heavily on Zarf for their deployments, has received requests to view manifests in a feedback session with their partners.
+
+### Goals
+
+<!--
+List the specific goals of the ZEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- View manifests or values files after Zarf variable templating and Helm templating have been applied.
+- View rendered package definition before create
+
+### Non-Goals
+
+<!--
+What is out of scope for this ZEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+- Accept a cluster source of the package for new functionality. Cluster source will continue to work for `zarf package inspect definition` and `zarf package inspect images`, but not for `zarf package inspect sbom`, `zarf package inspect manifests`, or `zarf package inspect values-files`
+- Print out multiple types of inspects in the same command. A user will not be able to print out the package definition and manifests in the same command. 
+- Support directly opening the SBOM in browser
+
+## Proposal
+
+<!--
+This is where you explain the specifics of the proposal. Provide enough detail
+for reviewers to clearly understand what you're proposing, but avoid including
+too many specifics like API designs or implementation details. Focus on the
+desired outcome and how success will be measured. The "Design Details" section
+below is for the real nitty-gritty.
+-->
+
+These commands will not have a confirm flag and will not prompt for optional components, package templates, or package variables. Users will be able to specify these values using flags, when applicable. None of these commands will run any Zarf actions.
+
+### zarf package inspect
+`zarf package inspect` will be deprecated and replaced by the five commands below
+#### zarf package inspect definition
+This will mirror behavior of the current `zarf package inspect <package>` command
```

Comment:
Nit, I'd avoid printing help or limit code fragments in enhancements. Only where strictly necessary (eg. API types) it seems reasonable. All other instances are better expressed in plain English. This ensures that implementation and proposal don't drift.

Reply:
Fair point, there would definitely be drift. I will edit to remove code blocks